Game/Src/Characters/Enemy.tscn

```diff
@@ -34,6 +34,7 @@ anims/Disappear = ExtResource( 3 )
 anims/RESET = SubResource( 3 )
 
 [node name="StatsEnemy" parent="." instance=ExtResource( 4 )]
-max_health = 0.04
+glory_win = 5.0
+glory_lose = 5.0
```

Comment:
should that be -5? I guess that in the code we have 2 functions add_glory and substract_glory that takes the params after the fight. But it seems easier to have a single function update_glory() that is called with a positive integer if winning and called with a negative integer if losing. What do you think?

Reply:
This is what I did, I have something like 'glory_increase' (lazy naming, 'glory_increment' would make more sense) that changes the glory by a given amount (positive or negative). I just do glory_increase(-glory_lose)